docs/getting-started-guides/aws/public.md

```diff
@@ -59,6 +59,7 @@ for i in $(ls .terraform/modules/*/Makefile); do i=$(dirname $i); make -C $i; do
 ### Provision the cluster infrastructure
 
 ```
```

Comment:
Can we just remove the file from the repo and this will just work when running terraform apply? so no need to manual step?